.github/workflows/publish.yaml

```
@@ -24,22 +24,39 @@ jobs:
         with:
           strip_v: true
 
+  get-previous-tag:
+    runs-on: ubuntu-22.04
+    outputs:
+      previous-tag: ${{ steps.previous-tag.outputs.tag }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Get previous tag
+        id: previous-tag
+        run: |
+          git fetch -a
+          tags=($(git tag --sort="-committerdate"))
```

reference: https://github.com/WyriHaximus/github-action-get-previous-tag/blob/master/main.js#L5C24-L5C43
```suggestion
          tags=($(git tag --sort="-creatordate"))
```